src/components/InactivePastMonthLearnersTable/index.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+
+import TableContainer from '../../containers/TableContainer';
+import { formatTimestamp, formatPercentage } from '../../utils';
+import EnterpriseDataApiService from '../../data/services/EnterpriseDataApiService';
+
+const InactivePastMonthLearnersTable = () => {
+  const tableColumns = [
+    {
+      label: 'Email',
+      key: 'user_email',
+      columnSortable: true,
+    },
+    {
+      label: 'Course Title',
+      key: 'course_title',
+      columnSortable: true,
+    },
+    {
+      label: 'Course Price',
+      key: 'course_price',
+      columnSortable: true,
+    },
+    {
+      label: 'Start Date',
+      key: 'course_start',
+      columnSortable: true,
+    },
+    {
+      label: 'End Date',
+      key: 'course_end',
+      columnSortable: true,
+    },
+    {
+      label: 'Passed Date',
+      key: 'passed_timestamp',
+      columnSortable: true,
+    },
+    {
+      label: 'Current Grade',
+      key: 'current_grade',
+      columnSortable: true,
+    },
+    {
+      label: 'Last Activity Date',
+      key: 'last_activity_date',
+      columnSortable: true,
+    },
+  ];
+
+
+  const formatPassedTimestamp = (timestamp) => {
+    if (timestamp) {
+      return formatTimestamp({ timestamp });
+    }
+    return 'Has not passed';
+  };
+
+  const formatTableData = enrollments => enrollments.map(enrollment => ({
+    ...enrollment,
+    last_activity_date: formatTimestamp({ timestamp: enrollment.last_activity_date }),
+    course_start: formatTimestamp({ timestamp: enrollment.course_start }),
+    course_end: formatTimestamp({ timestamp: enrollment.course_end }),
+    enrollment_created_timestamp: formatTimestamp({
+      timestamp: enrollment.enrollment_created_timestamp,
+    }),
+    passed_timestamp: formatPassedTimestamp(enrollment.passed_timestamp),
+    user_account_creation_timestamp: formatTimestamp({
+      timestamp: enrollment.user_account_creation_timestamp,
+    }),
+    has_passed: enrollment.has_passed ? 'Yes' : 'No',
+    course_price: enrollment.course_price ? `$${enrollment.course_price}` : '',
+    current_grade: formatPercentage({ decimal: enrollment.current_grade }),
+  }));
+
+  return (
+    <TableContainer
+      id="inactive-past-month-learners"
```

Comment:
If we combine the table components for the inactive learners for past week & past month into a single `InactiveLearnersTable`, we'll need to change the `TableContainer`'s id and className here depending on the `pastMonth` prop passed in (assuming we default to past week).